crates/astria-bridge-withdrawer/src/withdrawer/submitter/builder.rs

```diff
@@ -1,46 +1,92 @@
 use std::sync::Arc;
 
+use astria_core::primitive::v1::asset;
 use astria_eyre::eyre::{
     self,
     Context as _,
 };
-use tokio::sync::mpsc;
+use tokio::sync::{
+    mpsc,
+    oneshot,
+};
 use tokio_util::sync::CancellationToken;
 use tracing::info;
 
 use super::state::State;
-use crate::withdrawer::submitter::Batch;
+use crate::withdrawer::{
+    submitter::Batch,
+    SequencerStartupInfo,
+};
 
 const BATCH_QUEUE_SIZE: usize = 256;
 
+pub(crate) struct Handle {
+    startup_info_rx: Option<oneshot::Receiver<SequencerStartupInfo>>,
```

Comment:
don't think we need an option here - if the oneshot returns an error, then `recv_startup_info` should also error